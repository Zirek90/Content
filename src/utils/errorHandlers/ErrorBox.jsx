import React from "react";
import { ErrorContainter } from "../../styles/ContainerStyles";
import { ErrorBtn } from "../../styles/ButtonStyles";

export const ErrorBox = () => (
  <ErrorContainter>
    <p>Ops seems like there is some problem. Please reload page</p>
    <ErrorBtn onClick={() => window.location.reload()}>Reload page</ErrorBtn>
  </ErrorContainter>
);
