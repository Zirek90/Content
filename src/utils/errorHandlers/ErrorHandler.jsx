import React from "react";
import { ErrorExceptionWrapper } from "../../styles/ContainerStyles";

export const ErrorHandler = () => (
  <ErrorExceptionWrapper role="alert">
    <p>Could not download content</p>
  </ErrorExceptionWrapper>
);
