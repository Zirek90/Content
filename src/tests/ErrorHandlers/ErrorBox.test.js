import React from "react";
import renderer from "react-test-renderer";
import { ErrorBox } from "../../utils/errorHandlers/ErrorBox";

describe("ErrorBox", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<ErrorBox />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
