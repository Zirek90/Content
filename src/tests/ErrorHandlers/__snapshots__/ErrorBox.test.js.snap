// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorBox renders correctly 1`] = `
<div
  className="sc-eCApGN sc-jSFkmK cQhqkh"
>
  <p>
    Ops seems like there is some problem. Please reload page
  </p>
  <button
    className="sc-iCoHVE jVmhDW"
    onClick={[Function]}
  >
    Reload page
  </button>
</div>
`;
